app/webpacker/scripts/responsible-bodies-autocomplete.js

```diff
@@ -4,7 +4,8 @@ const initResponsibleBodiesAutocomplete = () => {
   try {
     const inputIds = [
       "#support-user-responsible-body-form-responsible-body-id-field",
-      '#school-search-form-responsible-body-id-field'
+      '#school-search-form-responsible-body-id-field',
+      '#support-ticket-academy-details-form-academy-name-field'
```

Comment:
@benilovj I'm thinking of renaming this file because I cannot see anything other than the id's that might make this functionality strictly for RB only.